main.py

```diff
@@ -1,9 +1,10 @@
 import wimp
 import json
+import erpcreds
 from multiprocessing.pool import ThreadPool
 
 if __name__ == "__main__":
-    session = wimp.get_session()
+    session = wimp.get_session(erpcreds)
```

Comment:
There should be an option to run it interactively so that it can be run locally.